azure-function/index.js

```diff
@@ -124,12 +129,16 @@ module.exports = async (context, req) => {
                 return;
             }
 
-            const sourceBranch = `refs/pull/${prNumber[1]}/head`;
+            const prNumber = match[2];
+            const sourceBranch = `refs/pull/${prNumber}/head`;
             const parameters = {
                 'pr.comment.id': comment.id,
```

Comment:
Why not pass the owner as part of the parms to the pipeline?  It just looks like a magic parameter in misc-helper.ts at the moment.  Of course, it this was to ever support a repo besides git, the repo name would need to be passed as well.

Reply:
> Why not pass the owner as part of the parms to the pipeline?

It is a magic parameter. And it kind of has to be: the Azure Pipeline needs to be installed _on that very same repository_. That is, when the Azure Function triggers GitGitGadget for git/git, it needs to know that pipelineId 13 is installed on git/git, and trigger that pipeline, if GitGitGadget should handle a PR at dscho/git, it needs to use the pipeline with the ID 12, and for regular PRs at gitgitgadget/git, the pipeline with ID 3 must be used.

The Azure Function therefore has to have intricate knowledge which pipelines were installed and which repository they target. I kinda hoped that the long commit message of https://github.com/gitgitgadget/gitgitgadget/pull/148/commits/8b08068e27aae1d27fb0195cbe3a776aae23b8bb was detailed enough?

Reply:
You know what, I'll just add a code comment about that to `index.js`.